70.md

```diff
@@ -0,0 +1,59 @@
+NIP-70
+======
+
+Protected Events
+----------------
+
+`draft` `optional`
+
+When the `"-"` tag is present, that means the event is "protected".
+
+A protected event is an event that can only be published to relays by its author. That allows the author to choose who can access their events (either by knowing in advance the relay policy with regards to reads or by means of an external relationship with the relay that is not specified here).
```

Comment:
> A protected event is an event that can only be published to relays by its **author**

What if optionally the `"-"` tag can have a list of pubkeys that are allowed to (re)publish the event (which may have been authored by someone else)?

It would be in line with for example a gift wrap that is authored by a random key but is meant to be "owned" by the receiving user (set on the `p` tag).

Reply:
That would leak a lot of metadata if used with gift wraps, defeating the entire purpose.

Reply:
only users authed by the p-tag should be able to download these events. Anyone can upload.

Reply:
>What if optionally the "-" tag can have a list of pubkeys that are allowed to (re)publish the event (which may have been authored by someone else)?

I thought of this too actually, I think it would work well. but figured that this NIP can always be extended to add this feature in the future so maybe just shelve it until theres an actual use case. most important thing right now is making rejection of events with `"-"` tag the default behavior.

Reply:
I thought that with e.g. `["-", "<author-pub>", "<receiver-pub>"]` the receiver could rebroadcast it to backup for example a local copy to another relay.

But yeah for gift wraps it doesn't make much sense as the receiver could re-wrap the seal. Just an idea but yeah haven't give it much thought.